framework/src/main/java/org/checkerframework/common/wholeprograminference/SceneToStubWriter.java

```diff
@@ -203,43 +223,63 @@ private static String formatArrayType(ATypeElement e, String arrayType) {
      *
      * @param e same as above, but can become null if scene-lib did not fill in the inner types,
      *     which happens when they do not have annotations
-     * @param arrayType same as above
+     * @param arrayTypes the array parts of the array type (i.e. the parts after the component
```

Comment:
Does this include annotations or not?
I'm wondering whether this is to be output as is or to be parsed and processed.

Reply:
It includes only explicitly-written annotations, not annotations that are going to be added. So it will be parsed so that annotations can be added at each level, and then mostly printed as-is.